data/page_text.yaml

```diff
@@ -18,72 +18,99 @@ gdp_processing_before: >-
+  each timestep, it will average the data within the given spatial extent. The processed data will
+  be a timeseries with one value for each timestep and for each spatial feature.
   
 gdp_algorithm_after: >-
   
-  [[Here is some info about the pic.]]
+  In the figure above, a user is requesting monthly temperature data from January to August 2015 
+  for a feature around Minnesota and Wisconsin. Each month has a grid of data associated with it 
+  which is clipped to the spatial feature. Each clipped grid is then averaged, leaving one 
+  value for each timestep. 
+  
 use_case_intro: >-
   # Extreme Storm Transposition Use-case
   
-  We are using an extreme storm transposition analysis to show off how the GDP works. Storm
+  We are using an extreme storm transposition analysis to show the GDP in action. Storm
   transpositions are a "what-if" 
   [water resource planning tool](https://toolkit.climate.gov/case-studies/using-demonstration-storms-prepare-extreme-rainfall) where actual extreme rain 
   events are simulated in different locations. Water resource managers can use this method to evaluate
   the impact a storm with similar intensity would have on their city and then implement risk-reducing
   measures where they are needed. In this use-case, we are recreating the 
   [Potter et al](http://onlinelibrary.wiley.com/doi/10.1111/1752-1688.12441/abstract)
-  extreme storm transposition for a storm in 2008 that impacted Baraboo, WI and moving it to Madison, WI.
+  extreme storm transposition for a storm in 2008 that impacted Baraboo, WI and seeing how a storm of that
+  magnitude might impact the Yahara River watershed near Madison. For this example, we are only going
+  to find the cumulative precipitation, and not continue the analysis by using that value for flood modeling.
 
 storm_transpose_before: >-
   ### Result of Storm Transposition
 
-  Here is a visualization product showing the extreme storm transposition from Baraboo to
-  Madison, Wisconsin.
+  Here is a visualization product showing the resulting cumulative precipitation from transposing the
+  extreme storm from Baraboo to the Yahara River watershed.  
   
 storm_transpose_after: >-
   
-  The data were selected and processed through the GDP. The following will break down how the 
-  GDP processed the data. The resulting impact on the watershed is out of scope for this 
-  visualization.
+  The storm dates were selected between June 2, 2008 to June 13, 2008. A shapefile of the Yahara River
+  watershed was downloaded using the [National Water Census Data Portal](https://cida.usgs.gov/nwc/). To
+  "transpose" the storm, the Yahara shapefile was first shifted north to the center of the extreme storm.
+  This new spatial feature was used as input to the GDP, along with the storm start and end dates, and 
+  the unweighted average algorithm was used to process this data. A dataset of average precipitation 
```

Comment:
Did we really use unweighted? Should use weighted. It's cooler and faster.

Reply:
I used the default for `geoknife`. Thought that was the unweighted average?

Reply:
Nope. It's weighted. "unweighted" is slower for long time series but works for very high spatial resolution because it doesn't store any intermediate processing data in memory. The price is that it does the spatial intersection for every time step (but forgets as it goes).